resource_types/core/401-paygcredit/paygcredit.oas.yaml

```
@@ -0,0 +1,254 @@
+---
+swagger: '2.0'
+info:
+  title: PAYG Credit
+  version: 'v0.7.1-2021'
+  license:
+    name: Angaza Nexus Data Model License (MIT License)
+    url: 'https://github.com/angaza/nexus-embedded/blob/master/LICENSE'
+    x-copyright: 'Copyright 2021 Angaza, Inc. All rights reserved.'
+schemes:
+  - coap
+consumes:
+  # Note: JSON provided to render examples using redoc, actual data transmitted
+  # is `application/vnd.ocf+cbor`
+  #- application/vnd.ocf+cbor
+  - application/json
+produces:
+  # - application/vnd.ocf+cbor
+  - application/json
+
+paths:
+  /nx/pc:
+    get:
+      summary: Get current PAYG credit state
+      description: |-
+        This resource indicates the remaining PAYG (pay-as-you-go) credit of
+        a specific device. Credit may be time-based or usage-based.
+
+        'modes' are implemented similar to the description of `oic.r.mode`
+        (https://www.oneiota.org/revisions/5310).
+
+        Any device may be 'independent'.
+        Controller-capable devices may be 'leading'.
+        Accessory-capable devices may be 'following'.
+        
+        A "Following" device has PAYG credit managed by another device which is
+        "Leading".
+
+        "Relaying" is not implemented in this release. A "Relaying" device
+        has PAYG credit managed by another device which is "Leading", but
+        a third device follows the "Relaying" device ('chaining' the credit
+        from device A to device C via device B, even though devices A and C
+        are not directly connected).
+
+        The default interface is 'oic.if.rw'. To retrieve the baseline OCF
+        information, it is necessary to query with 'if=oic.if.baseline'.
+
+        Application security constraints -
+        When used with Nexus Channel, the device POSTing must be a controller
+        with an active Nexus Channel Link to this device.
+
+        See also -
+        * angaza.com.nexus.channel.link_handshake - Information on Nexus channel link handshakes
+          
+      parameters:
+        # - $ref: '#/parameters/interface'
+      responses:
+        200:
+          description: "PAYG Credit specific information about this device."
+          schema:
+            $ref: '#/definitions/FullPAYGState'
+          x-example:
+            description: "x-example included only for automatic code generation."
+            rt:
+              - angaza.com.nexus.channel.core.paygcredit
```

```suggestion
              - angaza.com.nx.pc
```

This is what it is in the embedded dev repo.